packages/mobile/src/invite/saga.ts

```diff
@@ -5,7 +5,7 @@ import BigNumber from 'bignumber.js'
 import { Linking, Platform } from 'react-native'
```

Comment:
(Same as verification.ts)
Changes to this file just rewrite functions as generators, so that when they call functions that depend on zeroSyncMode, a selector can be used to read zeroSyncMode from state